documentation/guides/record-tests/README.md

```diff
@@ -31,7 +31,7 @@ The following sections provide detailed information about recording tests:
 
 ## Record Configurations
 
-You can record a test in any local browser. TestCafe detects browsers on the local computer and adds them to the **Record configuration** drop-down menu. Before recording a test, select the record configuration from the list.
+You can record tests in local browsers (see [Supported Browsers](../../references/supported-browsers.md)). TestCafe detects browsers on the local computer and adds them to the **Record configuration** drop-down menu. Before recording a test, select the record configuration from the list.
```

Comment:
Why don't we include the link into original text? It'll be shorter & easier to read. And in this context it's logical to lead people to the browsers we're talking about.

> You can record tests in [local browsers](../../references/supported-browsers.md).

Reply:
Usually, the link's text should correspond to the the topic's name. In your example, when a user clicks "local browsers", he/she expects to see a topic with the name "Local Browsers", but we have the "Supported Browsers" topic. 

Perhaps, it is an outdated rule.

Reply:
Ok, I've discussed this with Vasily, and we decide to use Natasha's variant.